pkg/controller/import-controller.go

```diff
@@ -220,12 +220,11 @@ func (r *ImportReconciler) Reconcile(_ context.Context, req reconcile.Request) (
 	volumeMode := cc.GetVolumeMode(pvc)
 	if volumeMode == corev1.PersistentVolumeBlock && pvc.GetAnnotations()[cc.AnnSource] == cc.SourceNone && pvc.GetAnnotations()[cc.AnnPreallocationRequested] != "true" {
 		log.V(1).Info("attempting to create blank disk for block mode, this is a no-op, marking pvc with pod-phase succeeded")
-		if pvc.GetAnnotations() == nil {
-			pvc.SetAnnotations(make(map[string]string, 0))
-		}
-		pvc.GetAnnotations()[cc.AnnPodPhase] = string(corev1.PodSucceeded)
-		if err := r.updatePVC(pvc, log); err != nil {
-			return reconcile.Result{}, errors.WithMessage(err, fmt.Sprintf("could not update pvc %q annotation and/or label", pvc.Name))
+		if pvc.Status.Phase == corev1.ClaimBound {
```

Comment:
So I am not sure about this. We purposely mark the PVC succeeded in the blank block case regardless of if it is bound or not. This is so that KubeVirt won't think it can't use the PVC. Once we have a consumer kubernetes will bind the PVC and everything resolves itself.

This was pre-populator logic. Now I don't really see how populators would affect blank block volumes at all, since there is no population. Maybe I am missing something.

Reply:
> This is so that KubeVirt won't think it can't use the PVC

KubeVirt should be fine with WFFC Pending PVCs/DV:
https://github.com/kubevirt/kubevirt/pull/9812

Reply:
Right, so I am trying to understand why we want this particular check here. I don't think we even want to use populators, unless maybe pre-allocation is set to true.

Reply:
The problem is if we stop using populators with this flow, we're back at the doppelganger pod solution

Reply:
For the blank volume without pre-allocation case, there is no need for the doppel ganger. It is 'populated' already considering there is nothing we have to do. It just gets bound when the VM starts, but it is still exactly the way we want it. With pre-allocation, I would expect our populator to attempt the pr-eallocation at the appropriate time.

Reply:
In my case, the PVC is not in bound state. It is in WFFC. 
If the issue is because I didn't enable HonorWaitForFirstConsumer feature gate, is there any side effect or behavior change of enabling this feature gate?

Reply:
So we've been going with this feature for quite a few releases now,
it basically means we respect WFFC binding by not creating any worker pods until PVCs get scheduled.

Is there any particular use case you have that would require bypassing WFFC behavior,
and bind PVCs randomly? Or will these blank disks be consumed by a VM?